kubeflow/kubeflow-training-operator/k8s/imagenet-pytorchjob.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: "kubeflow.org/v1"
+kind: "PyTorchJob"
+metadata:
+  name: "imagenet-pytorchjob"
+spec:
+  pytorchReplicaSpecs:
+    Master:
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "false"
+        spec:
+          containers:
+            - name: pytorch
+              image: navarrepratt/pytorch_dist_resnet50:9
+              command:
+                - "torchrun"
+                - "--nproc_per_node=8"
+                - "--nnodes=$(WORLD_SIZE)"
+                - "--node_rank=$(RANK)"
+                - "--master_addr=$(MASTER_ADDR)"
+                - "--master_port=$(MASTER_PORT)"
```

Comment:
@salanki - Yes, with `PyTorchJob` the master is doing compute as well. Not like the launcher in `mpijob`. In `PyTorchJob` the master and workers are pretty much the same except that `PyTorchJob` will set up the `MASTER_ADDR` and `MASTER_PORT` to point to the master.

Reply:
Ok, so that will give one extra node.